Integrations/MailSenderNew/MailSenderNew.py

```diff
@@ -333,19 +334,27 @@ def main():
     FROM = demisto.getParam('from')
     FQDN = demisto.params().get('fqdn')
     FQDN = (FQDN and FQDN.strip()) or None
+    TLS = demisto.getParam('tls')
     stderr_org = None
     try:
         if demisto.command() == 'test-module':
             stderr_org = swap_stderr(LOG)
             smtplib.SMTP.debuglevel = 1
-        SERVER = SMTP(demisto.getParam('host'), int(demisto.params().get('port', 0)), local_hostname=FQDN)
-        SERVER.ehlo()
+
         # TODO - support for non-valid certs
-        if demisto.getParam('tls'):
-            SERVER.starttls()
+        if TLS == 'SSL/TLS':
+            SERVER = SMTP_SSL(demisto.getParam('host'), int(demisto.params().get('port', 0)), local_hostname=FQDN)
+        else:
+            SERVER = SMTP(demisto.getParam('host'), int(demisto.params().get('port', 0)), local_hostname=FQDN)  # type: ignore
```

Comment:
When using `type: ignore` use the syntax: `# type: ignore[<error-name>]`  See: https://xsoar.pan.dev/docs/integrations/linting#mypy